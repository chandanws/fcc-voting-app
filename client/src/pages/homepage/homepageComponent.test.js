import React from "react";
import HomepageComponent from "./homepageComponent";
import { shallow } from "enzyme";

describe("<HomepageComponent />", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<HomepageComponent />);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
