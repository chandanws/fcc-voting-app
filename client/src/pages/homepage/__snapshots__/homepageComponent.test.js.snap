// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomepageComponent /> should render correctly 1`] = `
<div>
  <Typography
    component="h1"
    text="All polls"
  />
  <List
    modifiers={
      Array [
        "bordered",
      ]
    }
  >
    <Component
      modifiers={Array []}
    >
      <Button
        className=""
        component={[Function]}
        modifiers={
          Array [
            "align-left",
          ]
        }
        to="/polls/1"
      >
        Test
      </Button>
    </Component>
    <Component
      modifiers={Array []}
    >
      <Button
        className=""
        component={[Function]}
        modifiers={
          Array [
            "align-left",
          ]
        }
        to="/polls/2"
      >
        Test
      </Button>
    </Component>
    <Component
      modifiers={Array []}
    >
      <Button
        className=""
        component={[Function]}
        modifiers={
          Array [
            "align-left",
          ]
        }
        to="/polls/3"
      >
        Test
      </Button>
    </Component>
  </List>
</div>
`;
